src/components/Utils/orderResource.js

```diff
@@ -22,6 +22,14 @@ export const getFullOrderNumber = ({ poNumberPrefix = '', poNumberSuffix = '', p
 export const updateOrderResource = (order, mutator, changedProps) => {
   const clonedOrder = cloneDeep(order);
 
+  Object.assign(clonedOrder, changedProps);
+
+  return saveOrder(clonedOrder, mutator);
+};
+
+export const editOrderResource = (order, mutator, changedProps) => {
```

Comment:
it's totally similar names , editOrderResource  and updateOrderResource, so no clear purpose of introducing another similar method

Reply:
right. 
- `editOrderResource` is for editing PO (in this case PO number is parsed  to purePoNumber - numbers only with `initialValues` for form. And before saving PONumber = poNumberPrefix+poNumber+poNumberSuffix);
- `updateOrderResource` for all other cases such as open, approve, delete order - where no needed to change poNumber.

Reply:
Is that a problem to use updateOrderResource ? IMHO, idea of that function was to be a service function to call API.